CMakeLists.txt

```diff
@@ -208,7 +208,13 @@ configure_file(
   ${DCMTK_BINARY_DIR}/dcmtk.pc
   @ONLY
 )
+if (IS_ABSOLUTE "${CMAKE_INSTALL_LIBDIR}")
+  set(CMAKE_INSTALL_LIBDIR_PREFIX "${CMAKE_INSTALL_LIBDIR}")
+else()
+${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pkgconfig
+  set(CMAKE_INSTALL_LIBDIR_PREFIX "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
+endif()
 
 install(FILES "${DCMTK_BINARY_DIR}/dcmtk.pc"
-  DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pkgconfig
+  DESTINATION ${CMAKE_INSTALL_LIBDIR_PREFIX}/pkgconfig
```

Comment:
I think you can simplify these two hunks to just:

```
-   DESTINATION ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}/pkgconfig
+   DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig
```

CMake will ensure that CMAKE_INSTALL_LIBDIR gets prefixed as needed.